commands/pipelines/connect.js

```diff
@@ -0,0 +1,62 @@
+const cli = require('heroku-cli-util')
+const co = require('co')
+const api = require('../../lib/api')
+const KolkrabbiAPI = require('../../lib/kolkrabbi-api')
+const GitHubAPI = require('../../lib/github-api')
+const Validate = require('./setup/validate')
+const getGitHubToken = require('./setup/getGitHubToken')
+const getNameAndRepo = require('./setup/getNameAndRepo')
+const getRepo = require('./setup/getRepo')
+
+module.exports = {
+  topic: 'pipelines',
+  command: 'connect',
+  description: 'connect a github repo to an existing pipeline',
+  help: `Example:
+
+    $ heroku pipelines:connect example githuborg/reponame
+    Configuring pipeline... done`,
```

Comment:
This example isn't quite right, I think it should say...
```
$ heroku pipelines:connect example -r githuborg/reponame
```

Reply:
Fixed, good catch